src/js/components/Form/Form.js

```diff
@@ -93,7 +123,10 @@ const validate = (rule, fieldValue, formValue, format, messages) => {
         result = { message: result, status: rule.status };
       }
     }
+  } else if ('threshold' in rule) {
```

Comment:
Per @MikeKingdom 's comment, the default threshold never takes effect because this condition is never met.

```suggestion
  } else if (rule.max) {
```